tutorials/plugins/gdnative/gdnative-cpp-example.rst

```diff
@@ -28,7 +28,24 @@ Setting up the project
 ----------------------
 
 There are a few prerequisites you'll need:
-
+.. note::
+Names of Visual Studio's Components are translated using Google Translate, so they may be different from the ones you can find in Visual Studio
+
+- on Windows - Visual Studio with these installed components (you can easily download them from Visual Studio Instaler):
+- - Profile Tools for C++,
+- - Basic C++ Functions,
+- - Clang C++ Compiler for Windows (9.0.0),
+- - C ++ modules for build tools version 142 (x64 / x86 - experimental),
+- - Spectre Migrated Libraries (MSVC version 142 - VS 2019 - C ++ (Version 14.20)for x64/x86 architecture),
+- - MSVC version 142 - VS 2019 - C ++ compilation tools (version 14.20) for x64 / x86 architecture,
+- - C ++ / CLI support for build tools version 142 (14.20),
+- - MSM packages of the C ++ 2019 redistributable package,
+- - C ++ Clang-cl for build tools version 142 (x64 / x86),
+- - C ++ (version 14.20) ATL for build tools version 142 (x86 and x64),
+- - ATL version 142 - VS 2019 - C ++ compilation tools (version 14.20) for x64 / x86 architecture,
+- - MFC version 142 - VS 2019 - C ++ compilation tools (version 14.20) for x64 / x86 architecture,
+- - C ++ (version 14.20) ATL for build tools version 142 (x86 and x64),
+- - C ++ (version 14.20) MFC for build tools version 142 (x86 and x64),
```

Comment:
"C++ / CLI support" is C# related and is not required.
"C++ profile tools" not required.
"MFC and ATL support" are not required unless it's used directly in the project.
Example project do not have build config for cl-clang so it's also can't be used.

Visual Studio is updating constantly and specifying versions seems to be bad idea.

Selecting "Desktop development with C++" in "Workloads" tab of installer (in case VS 2017 - 2019) should be enough.